tests/fixture_valid.js

```diff
@@ -132,8 +146,20 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
     }
 
     for (const ch in fixture.availableChannels) {
-      if (usedChannels.indexOf(ch) === -1) {
-        result.warnings.push(`Channel '${ch}' defined but never used.`);
+      if (!usedChannels.includes(ch)) {
+        let isUsed = false;
```

Comment:
Please condense this into one statement: `const isUsed = 'switchingChannels' in fixture && Object.keys(fixture.switchingChannels).some(key => ...);`